update(blog)!: The future of privacy (#2874)
privacyguides/privacyguides.org: blog/posts/the-future-of-privacy.md

---
date:
    created: 2025-02-01T19:15:00
categories:
    - Opinion
authors:
    - em
description: Privacy is intrinsically intertwined with politics. Each change in governance can have serious effects on privacy rights and privacy tools, for better or for worse. Let's examine with concrete examples how politics affect legislations that can have an immense impact on the privacy tools and features we use.
schema_type: AnalysisNewsArticle
---

> # The Future of Privacy: How Politics Affect Laws Impacting Privacy Features 

Reviewer: This headline is unreadable, [Flesch-Kincaid score](https://capitalizemytitle.com/headline-analyzer/?headline=The%20Future%20of%20Privacy:%20How%20Politics%20Affect%20Laws%20Impacting%20Privacy%20Features) of 26.

```suggestion
# The Politics of Privacy: What New Laws Mean for Your Rights
```

```suggestion
# Privacy at Risk: How Political Shifts Shape Your Digital Rights
```

```suggestion
# The Future of Privacy: How Governments Shape Your Digital Life
```

Proponents of these regulations often refer to "age assurance processes" and suppose these processes to be undoubtedly secure. Anyone familiar with data security will understand how naive this approach is. I will not go into the details here, but you probably can already see how having each private website (or third-party processor) collect such sensitive information from each visitor is horrendous for privacy rights, and for data security as well. Of course these websites or third-party "age assurance processors" will unavoidably become large treasure trove for thieves, and its sensitive data [inevitably leaked or stolen](https://www.404media.co/id-verification-service-for-tiktok-uber-x-exposed-driver-licenses-au10tix/), sooner than later.

> Age Verification is one of the biggest threat to privacy online. Continuing in this direction could ultimately lead to the end of pseudoanonymous browsing. Additionally, this could also mean the end of your official ID having any value at all. After-all, what unique identification value does a piece of ID keep after it has been leaked in a thousand different data breaches? Maybe even one day bought on a [darknet market](https://en.wikipedia.org/wiki/Darknet_market) by a curious teenager in need of accessing some website...

Reviewer 2: ```suggestion
Age Verification is one of the biggest threats to privacy online. Continuing in this direction could ultimately lead to the end of pseudoanonymous browsing. Additionally, this could also mean the end of your official ID having any value at all. After-all, what unique identification value does a piece of ID keep after it has been leaked in a thousand different data breaches? Maybe even one day bought on a [darknet market](https://en.wikipedia.org/wiki/Darknet_market) by a curious teenager in need of accessing some website...
```

### Age Verification is already here, sadly

> Regrettably, this is not a hypothetical scare. Age Verification legislations have already passed in [Australia](https://www.thehindu.com/news/international/what-is-australias-online-safety-amendment-about-explained/article69026251.ece), in the [UK](https://www.ofcom.org.uk/online-safety/protecting-children/age-checks-to-protect-children-online/), as well as in [many U.S. states](https://action.freespeechcoalition.com/age-verification-resources/state-avs-laws/). It is also on the table federally in the [United States](https://www.eff.org/deeplinks/2024/10/eff-new-york-age-verification-threatens-everyones-speech-and-privacy), [Canada](https://www.eff.org/deeplinks/2024/09/canadas-leaders-must-reject-overbroad-age-verification-bill), [France](https://www.politico.eu/article/france-doubles-down-on-social-media-age-limit-at-15/), [Norway](https://www.euractiv.com/section/politics/news/norwegian-government-sets-15-year-age-limit-for-using-social-media/), and [Europe](https://digital-strategy.ec.europa.eu/en/funding/call-tenders-development-consultancy-and-support-age-verification-solution).

Reviewer: ```suggestion
Regrettably, this is not a hypothetical scare. Age Verification legislation has already passed in [Australia](https://www.thehindu.com/news/international/what-is-australias-online-safety-amendment-about-explained/article69026251.ece), in the [UK](https://www.ofcom.org.uk/online-safety/protecting-children/age-checks-to-protect-children-online/), as well as in [many U.S. states](https://action.freespeechcoalition.com/age-verification-resources/state-avs-laws/). It is also on the table federally in the [United States](https://www.eff.org/deeplinks/2024/10/eff-new-york-age-verification-threatens-everyones-speech-and-privacy), [Canada](https://www.eff.org/deeplinks/2024/09/canadas-leaders-must-reject-overbroad-age-verification-bill), [France](https://www.politico.eu/article/france-doubles-down-on-social-media-age-limit-at-15/), [Norway](https://www.euractiv.com/section/politics/news/norwegian-government-sets-15-year-age-limit-for-using-social-media/), and [Europe](https://digital-strategy.ec.europa.eu/en/funding/call-tenders-development-consultancy-and-support-age-verification-solution).
```

There is some [tenacious opposition](https://www.eff.org/deeplinks/2024/12/global-age-verification-measures-2024-year-review) to Age Verification policies from digital rights and free speech advocates. Unfortunately, there is also a strong push in support of Age Verification from the rapidly growing "age assurance" and identity verification [industry](https://avpassociation.com/thought-leadership/2025-the-year-of-implementation-for-age-assurance/), and from many governments worldwide **moving towards a surveillance state**.

> Again here, government values are deciding on digital features impacting our data privacy in disastrous ways. If you want to **take a stand against Age Verification**, you can join the [Stop Online ID Checks](https://www.stoponlineidchecks.org/) campaign from the nonprofit organization Fight for the Future.

Reviewer: ```suggestion
Again, government values are deciding on digital features that impact our data privacy in disastrous ways. If you want to **take a stand against Age Verification**, you can join the [Stop Online ID Checks](https://www.stoponlineidchecks.org/) campaign from the nonprofit organization Fight for the Future.
```

## The future of privacy

> There's a lot to be worrying about in today's privacy landscape. Unfortunately, recent political tendencies in the Western world make it difficult to stay optimistic. The trend towards authoritarian regimes and surveillance capitalism is bad news for the future of privacy around the globe.

Reviewer: ```suggestion
There's a lot to be worrying about in today's privacy landscape. Unfortunately, recent political tendencies in the Western world make it difficult to stay optimistic. The trend toward authoritarian regimes and surveillance capitalism is bad news for the future of privacy around the globe.
```

There is no question that privacy is intrinsically intertwined with politics, and can therefore never be politically neutral. The latest decisions taken by the new U.S. administration running full speed into deregulation and defunding, growing pressure in Europe to break end-to-end encryption in favor of a surveillance state, and invasive age verification policies to censor the web and collect even more data on every netizen is admittedly frightening. 

But one thing frightens me even more than all of this. One thing that could end privacy rights, forever. This threat to privacy is never far and always looming. 

This threat is giving up.

> Despite all the gloom menacing privacy rights, **privacy will never be dead as long as we stand up to defend it**. Governments might have the power to remove our privacy rights on paper and proclaim privacy features illegal. But the people have power to keep pushing for better privacy rights and to keep developing even more robust and more accessible privacy tools. 
> 
> We must continue to advocate loudly for privacy rights and for all human rights every chance we have. The fight for better privacy rights is only over when we give up.

Reviewer: ```suggestion
Despite all the gloom menacing privacy rights, **privacy will never be dead as long as we stand up to defend it**. Governments might have the power to remove our privacy rights on paper and proclaim privacy features illegal. But the people have the power to keep pushing for better privacy rights and to keep developing even more robust and more accessible privacy tools. 

We must continue to advocate loudly for privacy rights and *all* human rights every chance we have. The fight for better privacy rights is only over when we give up.
```
